test/models/ability_test.rb

```diff
@@ -954,15 +932,15 @@ def teardown
       tu = create_team_user team: t, user: u, role: 'collaborator'
       p = create_project team: t
       pm = create_project_media project: p
-      c = create_comment annotated: pm, annotator: u
-      c2 = create_comment annotated: pm
+      task1 = create_task annotated: pm, annotator: u
+      task2 = create_task annotated: pm
       with_current_user_and_team(u, t) do
         ability = Ability.new
-        assert ability.can?(:update, c)
-        assert ability.can?(:destroy, c)
-        assert ability.can?(:update, c2)
-        assert ability.can?(:destroy, c2)
-        c.destroy!
```

Comment:
Checks here against `comment` so we should remove not replace with another annotation.

Reply:
Question regarding this one, @melsawy. Since the test is to check that `collaborator should edit and destroy own annotation from trash but should not destroy respective log entry`, should we check that a collaborator is able to update and destroy an annotation?

Reply:
I think so, Jay!